backend/src/index.ts

```diff
@@ -61,6 +69,16 @@ const session_remove = async (sessionId: string) => {
   return true;
 };
 
+const sessionIdToUserId = async (sessionId: string) => {
+  const sessionData = query(sessions, where("sessionId", "==", sessionId));
+  const session = await getDocs(sessionData);
+
+  if (session.empty) {
+    return "guest";
```

Comment:
i think i'd prefer if this either:
a) returns `undefined`
b) returns a constant named `guest` that equals `undefined`